packages/elements/src/clock/index.ts

```diff
@@ -220,27 +241,49 @@ export class Clock extends ResponsiveElement {
   /**
    * Display the digital clock in 12hr format.
    */
-  @property({ type: Boolean, attribute: 'am-pm' })
+  @property({ type: Boolean, attribute: 'am-pm', reflect: true })
```

Comment:
Why reflect ?

Reply:
It actually is an idea from Alek. It should be reflect because it effects to UI.